src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style.includes(':'))
+    .reduce((acc, declaration) => {
+      const [property, value] = declaration
+        .split(':')
+        .map((item) => item.trim());
+
+      acc[property] = value;
```

Comment:
Assigning the value to the property in the accumulator object is correct. Ensure that the property names are unique, as duplicate properties will overwrite previous values.